### PR TITLE
TeamLBAC: Add scenario 6 to the docs

### PR DESCRIPTION
**Pull request by eleijonmarck**
**Why do we need this feature?**

We have questions from customers on what kind logs are expecting given a certain scenario. Specifically users are asking if the LBAC takes precedence.

**What is this feature?**
This adds a scenario for User A being Admin and part of team, and making sure that team's rules are only applied.

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md [hunk 1]
```diff
@@ -123,3 +123,14 @@ A user that is part of Team A will have access to logs that match `namespace="de
 A user that is part of Team A and part of Team B will have access to logs that match `namespace="dev"`.
 
 A user that is not part of Team A and part of Team B, that is `Editor` or `Viewer` will have access to all logs (due to the query permission for the user).
+
+### Task 6: User A is Admin and part of Team B
+
+We have team B, user A is part of Team B and has an `Admin`.
```
Comment (IevaVasiljeva) [inline docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md:129]: Nit: maybe specify that you're talking about org admin and not a team admin here?
```suggestion
We have team B, user A is part of Team B and has an `Admin` basic role.
```

File: docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md [hunk 2]
```diff
@@ -123,3 +123,14 @@ A user that is part of Team A will have access to logs that match `namespace="de
...
+
+- Team B has no roles assigned
+- Team B has Query permissions to Datasource Loki
+
+- Team B has rule `{ project_id="project-something-102461" }`
+
+User A may only query Datasource Loki where `{ project_id="project-something-102461" }` and no other logs on the data source.
```
Comment (IevaVasiljeva) [inline docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md:136]: nit: some grammar/style suggestions
```suggestion
- Team B has Query permissions to data source Loki

- Team B has a rule `{ project_id="project-dev" }`

User A may only access logs for data source Loki that match `{ project_id="project-dev" }` and no other logs on the data source.
```

**Review by IevaVasiljeva — COMMENTED** [review]
Added some suggestions. Maybe it's also worth moving [this note](https://github.com/grafana/grafana/blob/76e5be38db335c849219339ba3b5b6629b2be75f/docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md?plain=1#L93-L97) to the new section that you've added?

**Review by jdbaldry — APPROVED** [review]
We should generally avoid first person plural (like "we") when the page is actually referring to the user but this is consistent with the rest of the page so I don't think it makes sense to change that here.

**Pull request merged**